src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -618,6 +618,14 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           this._waitingEventTriggered = false;
           this._progressiveSources = [];
           this._loadPromise = null;
+          if (this._config.useSourceTag && this._videoElement) {
+            const source = this._videoElement.firstChild;
+            if (source) {
+              Utils.Dom.setAttribute(source, 'src', '');
```

Comment:
why do we need both setAttribute and removeAttribute ?

Reply:
It was considered best practice on the main video element and I implemented it on the source attribute as well

Reply:
Requires validation, @inbalvasserman can you please test it